CI/travis.osx.after_success.sh

```diff
@@ -134,6 +154,41 @@ if [ "${DEPLOY}" = "deploy" ]; then
 
       SHA256SUM=$(shasum -a 256 "${HOME}/Desktop/Mudlet-${VERSION}-${ARCH}.dmg" | awk '{print $1}')
 
+      # Create portable version (compressed .app bundle)
+      echo "=== Creating portable version ==="
+      PORTABLE_NAME="Mudlet-${VERSION}-${ARCH}-portable"
```

Comment:
Where does the `portable.txt` file get added here?

Reply:
line 258
```sh
touch "${PORTABLE_ZIP_DIR}/portable.txt"
```